taccsite_cms/templates/nav_cms.html

```diff
@@ -12,7 +12,5 @@
 <script type="module">
   import findLinksAndSetTargets from '/static/site_cms/js/modules/setTargetForExternalLinks.js';
 
-  const scopeElement = document.getElementById('cms-content');
-
-  findLinksAndSetTargets({ scopeElement });
+  findLinksAndSetTargets();
```

Comment:
This should not have only updated links in main content. It should also update links in header, main, and footer content.